spec/factories/topics.rb

```diff
@@ -1,6 +1,8 @@
 FactoryBot.define do
   factory :topic do
-    name "What is your secret superpower?"
+    sequence :name do |n|
+      "This is my topic number # #{n}"
```

Comment:
So there's a subtle error that can occur with sequences.
Be aware that when you have lots of specs you can easily end up with tens and even hundreds of instances being generated with a factory.

Consider the following expectation:
```ruby
expect(page).to have_text("This is my topic number 2")
```

This will pass for all of the following topics:
```
  "This is my topic number 2"
  "This is my topic number 20"
  "This is my topic number 21"
  "This is my topic number 22"
  ...
  "This is my topic number 200"
  "This is my topic number 201"
  "This is my topic number 202"
  ...
  "This is my topic number 220"
  ...
  etc
```

To avoid this problem, it is best to produce zero padded numbers.
For example:
```
  "This is my topic number 002"
  "This is my topic number 021"
  "This is my topic number 221"
```

You can do this with Ruby string formatting (`%`). See Ruby string object. This uses the `Kernel::sprintf` formatting options, so have a look at that too.